documentation/modules/overview/con-security-configuration-fips.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// overview/assembly-security-overview.adoc
+
+[id="security-configuration-fips_{context}"]
+= Federal Information Processing Standards (FIPS)
+
+[role="_abstract"]
+Federal Information Processing Standards (FIPS) are a set of security standards established by the US government to ensure the confidentiality, integrity, and availability of sensitive data and information that is processed or transmitted by information systems. 
+Strimzi automatically enables FIPS mode on the OpenJDK used in its container images when running on a FIPS-enabled Kubernetes cluster.
```

Comment:
Strimzi does not really do this. The OpenJDK does, Strimzi just _learned how to live with it. Maybe to avoid the details, we can say something like this?

```suggestion
OpenJDK used in Strimzi container images automatically enables FIPS mode when running on a FIPS-enabled Kubernetes cluster.
```